brainsets/descriptions.py

```
@@ -68,6 +68,65 @@ class SubjectDescription(temporaldata.Data):
+        try:
+            sex_normalized = Sex.from_string(sex)
+        except ValueError:
+            sex_normalized = Sex.UNKNOWN
+    elif isinstance(sex, int):
+        try:
+            sex_normalized = Sex(sex)
+        except ValueError:
+            sex_normalized = Sex.UNKNOWN
+
+    if species is None:
+        species_normalized = Species.UNKNOWN
```

Ruff (0.14.14)

[error] `Species` may be undefined, or defined from star imports

(F405)